### DISCUSSION TITLE
Virus Total Scan

### DISCUSSION THREAD

**Discussion by proxprox** [streamlink/streamlink#3449] [Q&A] [answered]
As a precaution, I scanned the windows .exe with virus total and it came up with some flags

https://www.virustotal.com/gui/file/999e3cb4c4cb1015d845a21c1429740f8e78b1cf3e65ca1c1696d5f803c748e1/detection 

just wanted you guys' opinion on this, thank you!:)

**Comment by gravyboat** [comment, marked as answer]
This [happens on the new releases occasionally](https://github.com/streamlink/streamlink/issues?q=is%3Aissue+virus+is%3Aclosed) and it's a false positive. There's nothing we can do about it unfortunately but you can read more on why it may be detected as a virus [here](https://github.com/streamlink/streamlink/issues/2661#issuecomment-541472929).

**Comment by bastimeyer** [comment]
Please take a look at already existing threads on the issue tracker before asking questions:
https://github.com/streamlink/streamlink/issues?q=virus

The whole concept of anti-virus software is utterly flawed and ridiculous, and this is a false positive. If you don't trust the installer, you can install Streamlink in a separate python environment via pip, etc.

See how the installer is built and deployed:
- https://github.com/streamlink/streamlink/blob/2.0.0/script/makeinstaller.sh
- https://github.com/streamlink/streamlink/blob/2.0.0/.github/workflows/main.yml#L71-L99
- https://github.com/streamlink/streamlink/runs/1594087601#step:7:1
- https://github.com/streamlink/streamlink/runs/1594102579?check_suite_focus=true#step:9:1

**Comment by proxprox** [comment]
Thanks so much for the responses! I figured it was a false positive, but I'm new to github so I didn't know if it was possible for someone to sneak code in without your knowledge.

Once again, thank you!
